### PR TITLE
FIX: Update removed `-autorecon-hemi` flag

### PR DESCRIPTION
**Pull request by mgxd**
Fixes https://github.com/nipreps/fmriprep/issues/2832

The `<no>label-exvivo-ec` is now removed, this substitutes it with  `<no>label-exvivo-ec-avg`, though I am not even sure this is necessary anymore?

FS changes from 2015... https://github.com/freesurfer/freesurfer/commit/939c3cbab4f81ef0b54dd2599a591f88dd528ac9

### CODE REVIEW

File: smriprep/interfaces/freesurfer.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ class _ReconAllInputSpec(fs.preprocess.ReconAllInputSpec):
             "parcstats2",
             "cortparc3",
             "parcstats3",
-            "label-exvivo-ec",
+            "label-exvivo-ec-avg",
```
Comment (effigies) [inline smriprep/interfaces/freesurfer.py:102]: We have some 5.3 and 6.0 tags. If this ever gets merged back into nipype, it would be good to not throw away 5.3 support needlessly.

```suggestion
            "label-exvivo-ec",  # 5.3
            "label-exvivo-ec-avg",  # 6.0+
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -737,6 +737,9 @@ jobs:
       - run:
           name: Install deps
           command: pip install -r docs/requirements.txt
+      - run:
+          name: Install package in editable mode
+          command: pip install -e .
```
Comment (effigies) [inline .circleci/config.yml:742]: Why is this needed?

Comment (effigies) [reply .circleci/config.yml:742]: It's fine, btw... I'm just skeptical of installing anything in editable mode in CI unless it's to test that editable mode works.

File: smriprep/interfaces/freesurfer.py (hunk 1 above)
Comment (mgxd) [reply smriprep/interfaces/freesurfer.py:102]: will this ignore using the `-<no>label-exvivo-ec` flag on FS6+? I did not realize the commented sections were still being used...

File: .circleci/config.yml (hunk 2 above)
Comment (mgxd) [reply .circleci/config.yml:742]: It isn't - the docs/requirement.txt was a mess of niworkflows / documentation dependencies, so cleaned it up

Comment (mgxd) [inline .circleci/config.yml:742]:
```suggestion
          name: Install package
          command: pip install .
```

File: smriprep/interfaces/freesurfer.py (hunk 1 above)
Comment (effigies) [reply smriprep/interfaces/freesurfer.py]: This just lists the valid steps. I'm not sure where `-<no>label-exvivo-ec` is being used. Could you point to the problem that's happening somewhere?

File: setup.cfg [hunk 3]
```diff
@@ -23,6 +23,7 @@ install_requires =
     matplotlib >= 2.2.0
     nibabel >= 4.0.1
     nipype >= 1.7.0
+    traits < 6.4
```
Comment (mgxd) [inline setup.cfg:26]: @effigies should we just cut a new nipype release?

File: smriprep/interfaces/freesurfer.py (hunk 1 above)
Comment (mgxd) [reply smriprep/interfaces/freesurfer.py]: So the problem is that `parcstats` is producing the command line:
```
-autorecon-hemi lh -nohyporelabel -lh-only -openmp 8 -subjid sub-XXXXXX -sd /out/sourcedata/freesurfer \
-notessellate -nosmooth1 -noinflate1 -noqsphere -nofix -nowhite -nosmooth2 -noinflate2 -nocurvstats \
-nosphere -nosurfreg -nojacobian_white -noavgcurv -nocortparc -nopial -nopctsurfcon -noparcstats \
-nocortparc2 -noparcstats2 -noparcstats3 -nolabel-exvivo-ec
```

and the `-<no>label-exvivo-ec` flag is no longer valid, causing an instant crash.

File: setup.cfg (hunk 3 above)
Comment (effigies) [reply setup.cfg:26]: Yes, I think so.

File: smriprep/interfaces/freesurfer.py (hunk 1 above)
Comment (effigies) [reply smriprep/interfaces/freesurfer.py]: Are you able to reproduce this in a debug session? I can't see why it would be happening, since `label-exvivo-ec` is only used [here](https://github.com/nipy/nipype/blob/4a91e88777465b435935d4dcaceff8a8345ba2dd/nipype/interfaces/freesurfer/preprocess.py#L1201), which is guarded by `if Info.looseversion() < LooseVersion("6.0.0"):`.

Comment (mgxd) [reply smriprep/interfaces/freesurfer.py]: Ah, I am able to reproduce this for nibabies, and see the problem - essentially `Info.looseversion() == 0.0.0` (since the developmental build-stamp.txt is not being correctly parsed), which causes this flag to be included when forcing `recon-all`.

I do not know why this is happening with fmriprep (https://github.com/nipreps/fmriprep/issues/2832) - we'll need to collect / run the `command.txt` and debug that.

Comment (mgxd) [reply smriprep/interfaces/freesurfer.py]: I believe so, but I have been using a prebuilt distribution instead of building FS (+ the infant module) from source.
